script/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 npm run build
-$(npm bin)/http-server . &
+$(npm bin)/http-server . > http.log 2>&1 &
```

Comment:
This was fixed in https://github.com/coala/gh-board/pull/12/files#diff-70d9b5fc8d0a3bcafbd21b0837258e25R4